Shorter description of prefix proofs (#21)
ietf-wg-keytrans/draft-protocol: draft-ietf-keytrans-protocol.md

hashes for any log entries whose timestamp was provided in `timestamps` but a
search proof was not provided in `prefix_proofs`.

If a log entry's timestamp is referenced multiple times, its timestamp MUST NOT
be repeated in `timestamps` but MUST only be added the first time. Additionally,
when a user advertises a previously-observed tree size in their request, log
entry timestamps that the user is expected to have retained are excluded from
`timestamps`.

`prefix_proofs` MAY contain multiple `PrefixProof` structures for the same log
entry. Users MUST verify that all `PrefixProof` structures corresponding to the
same log entry compute the same prefix tree root hash. There MAY be elements of

> `prefix_proofs` or `prefix_roots` that correspond to log entries whose timestamp
> is not included in `timestamps` but only if the user is expected to have
> retained the log entry's timestamp.

Reviewer: ```suggestion
`prefix_proofs` or `prefix_roots` that correspond to log entries whose timestamps
are not included in `timestamps` if the user is expected to have
retained the timestamps from a previous query response.
```
